recipes/mosquitto/2.x/conanfile.py

```diff
@@ -26,6 +26,7 @@ class Mosquitto(ConanFile):
         "cjson": [True, False],
         "build_cpp": [True, False],
         "websockets": [True, False],
+        "threading": [True, False],
```

Comment:
https://github.com/conan-io/conan-center-index/search?l=Python&q=threads&type=

The norm is

```suggestion
        "threads": [True, False],
```

Reply:
We used to follow same option names from the upstream, but I have to agree that keep an standard in CCI is much better. Also, threads is documented here: https://github.com/conan-io/conan-center-index/blob/master/docs/packaging_policy.md#recommended-feature-options-names

Reply:
I think it's best to align to the _project's own terminology_, which is called THREADING for one main reason:
Turning off THREADING does not mean not using threads and vice versa. It means not using any _threading related_ features such as mutex. This option is relevant for the mosquitto client library and not for the broker. This library offers two APIs - threaded and event loop. When setting THREADING to OFF we're saying we want the event loop API to **not use anything related to threading** such as mutexes because it may cause bugs or degrade performance.
It's crooked but that's the library and we're not changing it's code in thia MR.
So That's why I think calling the option "threads" may be confusing and counterproductive and its best to stick with mosquittos own terminology.

Reply:
Fair enough!